docs/en/entity_resolver_model_ner_model_pairs_edited.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: docs
+header: true
+title: Relation Extraction Model and NER Model Pairs
```

Comment:
- We need to change the title with `ENTITY RESOLVER MODEL AND NER MODEL PAIRS`. 
- This file and `docs/en/entity_resolver_model_ner_model_pairs.md` seem the same.